meerkat_frontend/src/files/pages/disease.html

```diff
@@ -208,22 +208,29 @@
 
     function drawDiseaseTable(varID, locID) {
 
+        function draw_top_table(category_name, title) {
```

Comment:
Maybe we use a more descriptive name for this function. What is top table?

Reply:
You got me here. Stale name, I've renamed it to reflect my best knowledge of what this reused piece of code does.